tests/integration/test_rids/test_rids_conf.py

```diff
@@ -1,7 +1,61 @@
-# Copyright (C) 2015-2021, Wazuh Inc.
-# Created by Wazuh, Inc. <info@wazuh.com>.
-# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
-
+'''
+copyright: Copyright (C) 2015-2021, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: The RIDS(Remote Identifiers) are the agent-manager remoted messages counter. Each message is identified by the
+       next possible number(identifier). Only the incoming messages with a valid RID(higher than previous) are allowed.
+       This functionality has a closing time value, which allows removing an agent's file handler when it does not send
+       a message during a period of time(five minutes by default).
+
+tier: 0
+
+modules:
+    - rids
+
+components:
+    - manager
+
+daemons:
+    - wazuh-remoted
```

Comment:
I would add wazuh-agentd
```suggestion
daemons:
    - wazuh-remoted
    - wazuh-agend
```

Reply:
https://github.com/wazuh/wazuh-qa/commit/93ae950e1f55e84b0ad12f9ef4dc99cd7e36b493 solves this